modules/AWS/nfs/outputs.tf

```diff
@@ -3,5 +3,5 @@ output "helm_release_nfs_service_ip" {
 }
 
 output "nfs_claim_name" {
-  value = var.shared_home_size == null ? null : kubernetes_persistent_volume_claim.shared-home-pvc[0].metadata[0].name
+  value = var.shared_home_size == null ? null : kubernetes_persistent_volume_claim.product_shared_home_pvc.metadata[0].name
```

Comment:
`var.shared_home_size == null ?`
Do we still need this condition?

Reply:
We don't, good catch!